src/commands/asset/delete.rb

```diff
@@ -6,24 +6,29 @@
 module Metalware
   module Commands
     module Asset
-      class Delete < Metalware::CommandHelpers::BaseCommand
-        include CommandHelpers::AssetHelper
+      class Delete < CommandHelpers::BaseCommand
+        include CommandHelpers::EnsureAssetExists
```

Comment:
Move the `include` under the private section. This way it is clearer that all these method will be private.